add fmt to libtorch_python.so

suo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#37560 add fmt to libtorch_python.so**

Differential Revision: [D21320059](https://our.internmc.facebook.com/intern/diff/D21320059)

suo committed:
add fmt to libtorch_python.so

[ghstack-poisoned]

suo committed:
Update on "add fmt to libtorch_python.so"


Differential Revision: [D21320059](https://our.internmc.facebook.com/intern/diff/D21320059)

[ghstack-poisoned]

zdevito commented on torch/csrc/jit/python/python_custom_class.cpp:
```diff
@@ -36,11 +38,10 @@ void initPythonCustomClassBindings(PyObject* module) {
+            fmt::format(
+                "Tried to instantiate class '{}', but it does not exist! "
+                "Ensure that it is registered via torch::jit::class_",
+                ns + "." + qualname));
```
use fmt and embed into string?

suo committed:
Update on "add fmt to libtorch_python.so"


Differential Revision: [D21320059](https://our.internmc.facebook.com/intern/diff/D21320059)

[ghstack-poisoned]

suo committed:
Update on "add fmt to libtorch_python.so"


Differential Revision: [D21320059](https://our.internmc.facebook.com/intern/diff/D21320059)

[ghstack-poisoned]
